vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/java/com/vaadin/flow/component/shared/internal/DisableOnClickController.java

```diff
@@ -77,9 +77,9 @@ public DisableOnClickController(C component) {
     public void setDisableOnClick(boolean disableOnClick) {
         this.disableOnClick = disableOnClick;
         if (disableOnClick) {
-            component.getElement().setProperty("disableOnClick", "true");
+            component.getElement().setAttribute("disableOnClick", "true");
```

Comment:
Technically could just set an empty string, but `Button` was specifically using a value of `true` before as well.

Reply:
As we are reverting to attributes, I would suggest using the name `disableonclick`, which, although is the same, better aligns with the outcome.

Reply:
Done